Models/Team.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Collections.Generic;
+using CribblyBackend.Interfaces;
+
+namespace CribblyBackend.Models
+{
+    public class Team
+    {
+        public string Name { get; set; }
+        public string Division { get; set; }
+        public List<Player> Players { get; set; }
+        public List<int> GameScores { get; set; }
+        public List<IGame> Games { get; set; }
```

Comment:
Are these referring to play in games specifically?

Reply:
This is a good point. I originally did intend this to just be play in games, but I wonder if it would make more sense to have separate members.